deploy/stacks/backend_stack.py

```diff
@@ -127,6 +132,12 @@ def __init__(
             **kwargs,
         )
 
+        # Create the SES Stack
+        ses_stack = None
+        if custom_domain == None and email_notification_sender_email_id == None and deploy_config.get_property("core.share_notifications.email.active") == True :
+            raise Exception("Cannot Create SES Stack For email notification as Custom Domain and Sender Email Id are not present. Either Disable Email Notification Config or add Custom Domain")
+        ses_stack = self.create_ses_stack(custom_domain, envname, kwargs, resource_prefix)
```

Comment:
In the if-clause in line 137 I think there is a problem. 
When deploy_config.get_property("core.share_notifications.email.active") == False, the if condition is not true, the exception is not raised and we end up creating the ses stack.

# Create the SES Stack
        ses_stack = None
        if deploy_config.get_property("core.share_notifications.email.active") == True :
            if custom_domain == None and email_notification_sender_email_id == None:
                raise Exception(".....")
            ses_stack = self.create_ses_stack(custom_domain, envname, kwargs, resource_prefix)